src/test/java/com/monumental/services/integrationtest/MonumentServiceIntegrationTests.java

```diff
@@ -45,61 +50,91 @@
     @Autowired
     TagService tagService;
 
-    /** bulkCreateMonumentsFromCsv Tests **/
+    public static String headers = "submittedBy,artist,title,date,materials,inscription,latitude,longitude,city,state,address,tags,reference,images";
+
+    public static Map<String, String> mapping = Map.ofEntries(
+        Map.entry("submittedBy", "submittedBy"),
+        Map.entry("artist", "artist"),
+        Map.entry("title", "title"),
+        Map.entry("date", "date"),
+        Map.entry("materials", "materials"),
+        Map.entry("inscription", "inscription"),
+        Map.entry("latitude", "latitude"),
+        Map.entry("longitude", "longitude"),
+        Map.entry("city", "city"),
+        Map.entry("state", "state"),
+        Map.entry("address", "address"),
+        Map.entry("tags", "tags"),
+        Map.entry("reference", "reference"),
+        Map.entry("images", "images")
+    );
+
+    public static List<String[]> parseCSVString(String csvRows) {
+        try {
+            CSVReader reader = new CSVReader(new StringReader(headers + "\n" + csvRows));
+            return reader.readAll();
+        } catch (IOException e) {
+            e.printStackTrace();
+            fail("An IOException occurred");
+            return new ArrayList<>();
+        }
+    }
 
-    @Test
-    public void testMonumentService_bulkCreateMonumentsFromCsv_NullCsvList() {
-        assertNull(this.monumentService.bulkCreateMonumentsFromCsv(null, false, null, null));
+    private MonumentBulkValidationResult validateCSV(String csvRows) {
+        try {
+            List<String[]> csvList = parseCSVString(csvRows);
+            return this.monumentService.validateMonumentCSV(csvList, mapping, null);
+        } catch (IOException e) {
+            e.printStackTrace();
+            fail("An IOException occurred");
+            return new MonumentBulkValidationResult();
+        }
```

Comment:
What's the reason for these two `static` methods in here?

Reply:
Basically I had a bunch of code copy and pasted in the tests so I broke them out into these functions